crypto/Pos/Binary/Crypto.hs

```diff
@@ -126,45 +128,43 @@ BiMacro(C.EncShare, encShareBytes)
 -- Signing
 ----------------------------------------------------------------------------
 
-instance Bi Ed25519.PointCompressed where
-    encode (Ed25519.unPointCompressed -> k) = encode k
-    decode = Ed25519.pointCompressed <$> decode
+encodeXSignature :: CC.XSignature -> E.Encoding
+encodeXSignature a = E.encodeBytes $ CC.unXSignature a
```

Comment:
Previously it was:
```
instance Bi CC.XSignature where
    encode (CC.unXSignature -> bs) = encode bs
```
, but now you are using `E.encodeBytes` instead of `encode`. Why? Even if it's equivalent, I think it's better to leave `encode`.
It also applies to some other places.

Reply:
`encode` it is than